frontend/src/services/ApiServiceFactory.ts

```diff
@@ -63,6 +64,10 @@ export default class ApiServiceFactory {
                 FAMApplicationAdminApi,
                 adminManagementBaseURL
             ),
+            applicationsApi: this.createApiInstance(
+                FAMApplicationsApi,
```

Comment:
maybe doesn't matter, but if we want to use alias, shall we change to use alias in all places? also at line 22

Reply:
All these API names were actually generated from client-code-gen.
Frontend only imports them.
The reason I put the comment and use alias on line 7 was both `fam-admin-mgmt-api` and `fam-app-acsctl-api` all contains generated api(s) called "**FAMApplicationsApi**" and so both conflicts each other when import into the same file.
I prefer to stay the name they are generated from, however, in this case it happens to be the same so I had to choose using alias to avoid the conflict (in this case, I intended to let `fam-admin-mgmt-api` have the original name). This change will not affect external code that uses both api(s).
I can ask you tomorrow, maybe you have better idea or maybe I misunderstand your thought.

Reply:
Hi Ian, now I got it, all good, thanks!!